server.py

```diff
@@ -0,0 +1,59 @@
+#python -m uvicorn server:app --reload
+
+import sqlite_helpers
+from fastapi import FastAPI, Request, HTTPException
+from fastapi.responses import RedirectResponse, HTMLResponse
+from datetime import datetime
+
+from constants import HttpResponse, http_code_to_enum
```

Comment:
a good order for python imports goes:
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
```suggestion
import datetime

from fastapi import FastAPI, Request, HTTPException
from fastapi.responses import RedirectResponse, HTMLResponse

import sqlite_helpers
from constants import HttpResponse, http_code_to_enum

```

see the pep 8 python standard https://peps.python.org/pep-0008/#imports